ToyBox/classes/MainUI/EnhancedUI/BulkSell.cs

```diff
@@ -0,0 +1,124 @@
+﻿using ModKit.Utility;
+using System;
+using System.Linq;
+using ToyBox.classes.Models;
+using UnityEngine;
+using static ModKit.UI;
+
+namespace ToyBox
+{
+    internal static class BulkSell
+    {
+        private static readonly BulkSellSettings _settings = Main.Settings.bulkSellSettings;
+
+        public static void OnGUI()
+        {
+            void bonusItemOptions(string itemTypeName, string bonusType, ref int enchantLevel, ref int stackSize)
```

Comment:
Please capitalize method names

Reply:
Even locally scoped functions like this?